### DISCUSSION TITLE
Tijdreeksen opvragen met iMOD plugin geeft error

### DISCUSSION THREAD

**Discussion by SiebeBosch** [Deltares/Ribasim#510] [Q&A] [answered]
Ik heb het basic model (uit example-models, prerelease juni 2023) doorgerekend met de CLI. Daarna heb ik het model ingeladen in QGis (plugins ribasim en iMod geïnstalleerd van de prerelease van juni 2023). Vervolgens heb ik Node Output geassocieerd met met basin.arrow en Edge Output met flow.arrow.

Tenslotte in het lagenpaneel het model geselecteerd (mapje basic).
Als ik daarna op de timeseries knop van de iMOD plugin klik krijg ik de volgende error:

load_arrow_data
              df = read_arrow(arrow_path)
              File "C:\Users/SiebeBosch/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\imodqgis\arrow\reading.py", line 7, in read_arrow
              layer = dataset.GetLayer(0)
             AttributeError: 'NoneType' object has no attribute 'GetLayer'

**Comment by visr** [comment]
Hi Siebe,

What version of QGIS are you using? Older versions of QGIS don't come with arrow support. I believe we need at least QGIS 3.28.
If that's not it, could you perhaps try downloading the latest builds of the QGIS plugin from https://deltares.github.io/Ribasim/ and use that together with up to date example models: [example-models.zip](https://github.com/Deltares/Ribasim/files/12370000/example-models.zip).

**Reply by SiebeBosch** [reply]
Hi Martijn!

I am  using QGis version 3.24 and will upgrade first! 

Will let you know asap!
Siebe

**Comment by visr** [comment, marked as answer]
This error actually appeared when opening the timeseries widget whilst there is no Arrow file present. The error is fixed in iMOD QGIS plugin 0.5.1.
